src/routes/+page.svelte

```diff
@@ -0,0 +1,100 @@
+<script lang="ts">
```

Comment:
Put register page under the correct path

Reply:
Done.